changedetectionio/templates/edit.html

```diff
@@ -172,6 +172,16 @@
                             <span class="pure-form-message-inline">Good for websites that just move the content around, and you want to know when NEW content is added, compares new lines against all history for this watch.</span>
                         </div>
                     </fieldset>
+                    <fieldset>
+                        <div class="pure-control-group">
+                            <label for="trigger-type">Filter and restrict change detection of content to</label>
+                            {{ render_checkbox_field(form.trigger_add, class="trigger-type") }}
+                            {{ render_checkbox_field(form.trigger_del, class="trigger-type") }}
+                            <span class="pure-form-message-inline">
+                                Filters the change-detection of this watch to only this content change.
```

Comment:
```suggestion
                                Filters the change-detection of this watch to only this type of content change. <strong>replacements</strong> (neither additions or deletions) are always included.
```

Reply:
What do you think?

Reply:
Capitalize the “r” in replacements and then it looks good to me

Reply:
Resoved via [c80f463](https://github.com/dgtlmoon/changedetection.io/pull/749/commits/c80f46308aa87168142249c413e915963eb70bda)

Reply:
Cool thanks! I think it makes way more sense now, trying to think of situations that might confuse people (end up in more issues in the queue :) )

Reply:
Totally get it!